@types/twin.d.ts

```diff
@@ -7,31 +7,26 @@
 // ─── MODULE AGUMENTATION IN TYPESCRIPT TO EXTEND EXISTING TYPES WITH ADDITIONAL PROPERTIES
 //
 
+// twin.d.ts
 import "twin.macro";
-import styledImport, { CSSProp, css as cssImport } from "styled-components";
+import styledImport from "@emotion/styled";
+import { css as cssImport } from "@emotion/react";
+
+// The css prop
+// https://emotion.sh/docs/typescript#css-prop
+import {} from "@emotion/react/types/css-prop";
 
 declare module "twin.macro" {
     // The styled and css imports
     const styled: typeof styledImport;
     const css: typeof cssImport;
 }
 
-declare module "react" {
-    // The css prop
-    interface HTMLAttributes<T> extends DOMAttributes<T> {
-        css?: CSSProp;
-    }
-    // The inline svg css prop
-    interface SVGProps<T> extends SVGProps<T> {
-        css?: CSSProp;
-    }
-}
-
-// The 'as' prop on styled components
-declare global {
-    namespace JSX {
-        interface IntrinsicAttributes<T> extends DOMAttributes<T> {
-            as?: string;
-        }
-    }
-}
+// // The 'as' prop on styled components
```

Comment:
Did you mean to leave this commented-out code here? If so, leave it in. If not, take it out.